### PR TITLE
[DOC] Edit backuprestore content initial

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**
Initial edit of MKE 4 backuprestore content, including text changes and content restructuring/granulation.

### CODE REVIEW

**Review by tppolkow — CHANGES_REQUESTED** [review]

**Review by tppolkow — APPROVED** [review]
LGTM

File: content/docs/reference/backuprestore/backup-restore.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+| storage_provider.in_cluster_options.distributed            | Indicates whether to run MinIO in distributed mode.                  | true, false         |   false   |
+| storage_provider.external_options.provider                 | Name of the external storage provider. Note that AWS is currently the only available option.                            | aws                 |    aws    |
+| storage_provider.external_options.bucket                   | Name of the pre-created bucket to use for backup storage.                            | ""                  |     ""    |
+| storage_provider.external_options.region                   | Region in which the bucket exists.                                                   | ""                  |     ""    |
+| storage_provider.external_options.credentials_file_path    | Path to Credentials File.                                                            | ""                  |     ""    |
+| storage_provider.external_options.credentials_file_profile | Profile in the Credentials file to use.                                              | ""                  |     ""    |
+
+## Create backups and restores
```
[the remark this answers is not in the document]
Comment (KoryKessel-Mirantis) [reply content/docs/reference/backuprestore/backup-restore.md:48]: I discussed the word 'restore' in this context with @tppolkow, and he informed me that **restore** is the name of the underlying Kubernetes object. Alongside that, I did a bit of Internet searching and found the use of **restore** as a verb in reference to software systems is quite common. So all of that said, I suggest we meet in the middle and insert **perform** but keep **restores**.

File: content/docs/reference/backuprestore/external.md [hunk 2]
```diff
@@ -1,55 +1,67 @@
...
+## Create an external backup
+
+To create an external backup, run:
 
 ```shell
 mkectl backup create --name aws-backup
 ```
 
-Output:
+Example Output:
```
[the remark this answers is not in the document]
Comment (KoryKessel-Mirantis) [reply content/docs/reference/backuprestore/external.md:63]: I suggest we stick with **Example output

File: content/docs/reference/backuprestore/in_cluster.md [hunk 3]
```diff
@@ -1,8 +1,20 @@
-# Backup and Restore Using In Cluster Storage Provider
-By default, MKE will store backups and restores using the in cluster storage provider (MinIO). The below instructions assume you have created a cluster and applied a blueprint with the default MKE backup configuration.
+# Backup and Restore using the In-Cluster Storage Provider
+By default, MKE 4 stores backups and restores using the in-cluster storage
+provider, the [MinIO addon](https://microk8s.io/docs/addon-minio).
 
-## Usage
-Backups can be created using `mkectl backup create --name <name>`.
+>The offered instructions assume that you have created a cluster and
```
[the remark this answers is not in the document]
Comment (KoryKessel-Mirantis) [reply content/docs/reference/backuprestore/in_cluster.md:5]: In my opinion, yes, this type of content should usually be set apart from the main textual flow in some manner.

**Review by ogusarenko — APPROVED** [review]

**Pull request merged**